README.md

```diff
@@ -923,6 +925,57 @@ Guidelines:
 
 References for other CLIs which collect analytics are [Angular CLI](https://angular.io/analytics), and [Next.js](https://nextjs.org/telemetry) CLI.
 
+# 9 Appendix: CLI Frameworks
+
+### 9.1 CLI Frameworks Table
+
+<!-- markdownlint-disable -->
+<table>
```

Comment:
This is great but can you convert the HTML table into Markdown syntax for tables? See here: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables